modules/javafx.graphics/src/main/java/com/sun/prism/impl/shape/DMarlinPrismUtils.java

```diff
@@ -44,31 +44,29 @@
 
     private static final boolean FORCE_NO_AA = false;
 
-    static final float UPPER_BND = Float.MAX_VALUE / 2.0f;
-    static final float LOWER_BND = -UPPER_BND;
-
-    static final boolean DO_CLIP = MarlinProperties.isDoClip();
-    static final boolean DO_CLIP_FILL = true;
+    // slightly slower ~2% if enabled stroker clipping (lines) but skipping cap / join handling is few percents faster in specific cases
+    static final boolean DISABLE_2ND_STROKER_CLIPPING = true;
 
     static final boolean DO_TRACE_PATH = false;
 
+    static final boolean DO_CLIP = MarlinProperties.isDoClip();
+    static final boolean DO_CLIP_FILL = true;
```

Comment:
This was there before as well, but what is the purpose? It seems to be only used on a single place. Should this be a boolean that can be set via a property?

Reply:
I like constants that are inlined well by hotspot. Moreover the clip fill flag helped me in Marlin 0.8 to handle properly the even/odd rule.
It could be removed but I think it is more obvious like this.